src/core/Link/index.tsx

```diff
@@ -1,12 +1,14 @@
-import { Link as RawLink } from "@mui/material";
 import React, { ForwardedRef, forwardRef } from "react";
 import { LinkProps, StyledLink } from "./style";
 
 /**
  * @see https://v4.mui.com/components/links/
  */
 const Link = forwardRef(
-  (props: LinkProps, ref: ForwardedRef<HTMLAnchorElement>) => {
+  <C extends React.ElementType>(
```

Comment:
this is also needed for supporting `component` prop in addition to:

```ts
// (thuang): Support `component` prop
// https://stackoverflow.com/a/66123108
export type LinkProps<C extends React.ElementType = "a"> = RawLinkProps<
  C,
  { component?: C }
> &
  StyleProps & {
    sdsStyle?: "default" | "dashed";
  };
```

Reply:
Maybe @masoudmanson was trying to solve the `component` prop issue with casting Link as `RawLink`?

Reply:
Thank you @tihuan for fixing this one, There was a linter error on the link component and I had to export it as the type of Link back then. Didn't know it would cause problems later. 💯